### PR TITLE
add legacy search monitor job deployment script

### PR DESCRIPTION
**Pull request by jinguiy**

### CODE REVIEW

File: scripts/monitor-mcm/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
+    echo Usage: $0 \<DOCK_REGISTRY\> \<NAME_SPACE\> \<SA\>
+    echo You must specify docker registry for docker image of monitor job, namespace of legacy search and service account to deploy the monitor job and a build number.
```
Comment (stacyped) [inline scripts/monitor-mcm/build.sh:5]: Edit to "To deploy the monitor job and create? a build number, you must specify the docker registry for the docker image of the monitor job, namespace of legacy search, and service account  "

Or maybe

"You must specify the docker registry for the docker image of the monitor job, namespace of legacy search, and service account to deploy the monitor job and a build number."

File: scripts/monitor-mcm/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# overview
+The script to monitor legacy search and recover the legacy search from aggregator hang or redisgraph can't response issues.
```
Comment (stacyped) [inline scripts/monitor-mcm/README.md:2]: maybe an edit like "This is the script to monitor legacy search and recover the legacy search from aggregator hang. This enables the redisgraph to respond to issues."
OR
Without this script, redisgraph can't respond to issues.

File: scripts/monitor-mcm/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# steps to deploy the legacy search monitoring job
```
Comment (stacyped) [inline scripts/monitor-mcm/README.md:4]: Steps

File: scripts/monitor-mcm/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+1. Prepare one service account with docker registory pull and following pod authority
```
Comment (stacyped) [inline scripts/monitor-mcm/README.md:5]: Edit to "Prepare one service account with Docker registry pull and the following pod authority: "

File: scripts/monitor-mcm/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/log
+  - pods/exec
+  verbs:
+  - '*'
+``` 
+2. run build.sh command to create job
```
Comment (stacyped) [inline scripts/monitor-mcm/README.md:16]: Edit to "Run the build.sh command to create a job: "

File: scripts/monitor-mcm/build.sh (hunk 1 above)
Comment (jinguiy) [reply scripts/monitor-mcm/build.sh]: Updated.

File: scripts/monitor-mcm/README.md (hunk 2 above)
Comment (jinguiy) [reply scripts/monitor-mcm/README.md]: aggregator hang and redisgraph can't response are two different issues. this job can monitor the search and recover it from these two issues. I updated the overview. would you pls help check whether it is ok? thanks!

File: scripts/monitor-mcm/README.md (hunk 3 above)
Comment (jinguiy) [reply scripts/monitor-mcm/README.md]: Updated.

File: scripts/monitor-mcm/README.md (hunk 4 above)
Comment (jinguiy) [reply scripts/monitor-mcm/README.md]: Updated

File: scripts/monitor-mcm/README.md (hunk 5 above)
Comment (jinguiy) [reply scripts/monitor-mcm/README.md]: Updated.

**Pull request merged**